CHANGELOG.md

```diff
@@ -36,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Removed the `libudev` feature (#742)
 - The `FirmwareImage` trait no longer includes the `segments_with_load_addresses` function (#796)
+- Removed the `flasher::parse_partition_table` function (#798)
```

Comment:
Maybe we should mention that it now lives under the `cli` mod?

Reply:
We could, I don't really want to encourage people to use that module in their own code though.